lib/mongodb/mongo.go

```diff
@@ -0,0 +1,22 @@
+package mongodb
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+func NewConnection(clientOptions *options.ClientOptions) (*mongo.Client, error) {
```

Comment:
I think it should return an error not panic in this function because we know when an error happened